src/Utilities/modif_matrix.jl

```diff
@@ -0,0 +1,63 @@
+function MOI.delete(v::MatrixOfConstraints, ci::MOI.ConstraintIndex)
```

Comment:
Why are these functions in a different file? They aren't tested.

Reply:
Oops, this file shouldn't have been pushed